kolibri_explore_plugin/views.py

```diff
@@ -77,7 +77,7 @@ def _get_file(self, app, path):
 
 
 class AppView(AppBase):
-    @metadata_cache
+    @method_decorator(metadata_cache, name="dispatch")
```

Comment:
I think you should squash this with the original `cache_forever` change.

Reply:
Good point!